tests/src/Kernel/fixtures/markup_rendering_patterns/banner.yml

```diff
@@ -251,6 +323,41 @@ image_page_banner_centered_with_link_full_width:
       'p': "Innovation, economy, environment and geopolitics"
     equals:
       'a': "Subscribe"
+image_page_banner_centered_with_link_full_width_with_content_classes:
+  render:
+    '#type': pattern
+    '#id': banner
+    '#variant': 'image'
+    '#fields':
+      settings:
+        'hero': false
+        'shade': false
+        'full_width': true
+        'centered': true
+      'title': "EU Budget for the future"
+      'description': "Innovation, economy, environment and geopolitics"
+      'link':
+        'label': "Subscribe"
+        'path': "https://example.com"
+      'image': "https://placeimg.com/1000/500/arch"
+      'content_classes': "content-class my-2"
+  assertions:
+    count:
+      'div[data-drupal-selector=banner]': 1
+      'div.bcl-banner.bg-lighter.text-dark': 1
```

Comment:
Add classes text-center full-width which come from activating the full_width and centered settings.